server/src/test/java/org/opensearch/action/admin/cluster/node/stats/NodeStatsTests.java

```diff
@@ -718,7 +719,8 @@ public static NodeStats createNodeStats(boolean remoteStoreStats) {
                 randomBoolean() ? new PendingClusterStateStats(randomInt(), randomInt(), randomInt()) : null,
                 randomBoolean()
                     ? new PublishClusterStateStats(randomNonNegativeLong(), randomNonNegativeLong(), randomNonNegativeLong())
-                    : null
+                    : null,
+                randomBoolean() ? new ClusterStateStats() : null
```

Comment:
new stats tests

Reply:
Added. 
Structure tests are added in NodeStatsTests.
Overall stats tests are added in ClusterManagerService implementation test file.
RemotePersistenceStats tests are added in RemoteClusterStateService itself.